src/Conversions/include/BipedalLocomotion/Conversions/ManifConversions.h

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file Conversions.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_MANIF_CONVERSIONS_H
+#define BIPEDAL_LOCOMOTION_MANIF_CONVERSIONS_H
+
+#include <iDynTree/Core/Transform.h>
+#include <iDynTree/Core/EigenHelpers.h>
+#include <manif/manif.h>
+
+
+namespace BipedalLocomotion
+{
+namespace Conversions
+{
+    /**
+     * @brief Convert rotation matrix and translation vector to manif SE3 object
+     *
+     * @param rotation reference to 3x3 Eigen matrix
+     * @param translation reference of 3x1 Eigen matrix
+     * @return pose as manif SE3 object
+     */
+    template <class Scalar>
+    manif::SE3<Scalar> toManifPose(const Eigen::Matrix<Scalar, 3, 3>& rotation,
+                                   const Eigen::Matrix<Scalar, 3, 1>& translation)
```

Comment:
Also these probably should be ``Ref`` right?

Reply:
With Refs, I was not able to pass template parameters. This is why I did an overload with one using Refs of Matrix doubles and another without using Refs.

Reply:
Ah ok. Fine for me then